src/Sarif/Core/SarifLog.cs

```diff
@@ -86,7 +150,7 @@ public void Save(string sarifFilePath)
         /// <param name="stream">Stream to write SARIF to</param>
         public void Save(Stream stream)
         {
-            using (StreamWriter streamWriter = new StreamWriter(stream))
+            using (var streamWriter = new StreamWriter(stream))
```

Comment:
>var [](http://example.com/codeflow?start=19&length=3)

aren't you using the single line using convention? if so, feel free to switch to it when you modify code like this. It's a nice practice to scrub an entire file if you make a single change of this kind.

there's a case for keeping things consistent (i.e., don't make a change) if the costs of updating everything are too high.